### PR TITLE
improve log levels

### PR DESCRIPTION
**Pull request by JssDWt**
Logs currently contain unuseful sections like below, cluttering the logs.
This PR sets the default log level to `info`, and all sdk related loglevels to `debug`.

```
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.170036Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.200344Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.218030Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.253748Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.272098Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.307360Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.329893Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.356449Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.369552Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.398529Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.412128Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.442880Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.452913Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.483022Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.494048Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.531994Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.543038Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.578944Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.589314Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.620291Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.629984Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.663966Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.673143Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.708896Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.720016Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.759743Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.772713Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.805306Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.814986Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.848213Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.857246Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.885252Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.893643Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.929084Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.941174Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.977456Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:08.995694Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:09.030732Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:09.060805Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:09.090097Z) : received frame=Data { stream_id: StreamId(3) }
[BreezSdk] {CONFIG} (2024-10-07T14:17:09.128401Z) : received frame=Data { stream_id: StreamId(3) }
```

### CODE REVIEW

**Comment by erdemyerebasmaz** [comment]
Related: #874

File: libs/sdk-common/src/utils/rest_client.rs [hunk 1]
```diff
@@ -40,7 +40,7 @@ pub async fn get_and_log_response(
     let response = get_reqwest_client()?.get(url).send().await?;
     let status = response.status();
     let raw_body = response.text().await?;
-    debug!("Received response, status: {status}, raw response body: {raw_body}");
+    trace!("Received response, status: {status}, raw response body: {raw_body}");
```
Comment (roeierez) [inline libs/sdk-common/src/utils/rest_client.rs:43]: perhaps log the status only on debug? It is very helpful to understand if the chain service failed or not.

**Review by roeierez — CHANGES_REQUESTED** [review]

File: libs/sdk-core/src/breez_services.rs [hunk 2]
```diff
@@ -1820,11 +1820,9 @@ impl BreezServices {
...
             .parse_filters(
                 r#"
-                debug,
-                breez_sdk_core::input_parser=warn,
-                breez_sdk_core::backup=info,
-                breez_sdk_core::persist::reverseswap=info,
-                breez_sdk_core::reverseswap=info,
+                info,
+                breez_sdk_core=debug,
+                sdk_common=debug,
```
Comment (ok300) [inline libs/sdk-core/src/breez_services.rs:1825]: NIT: now that the root logger is set to `info`, do you think adding
```
breez_sdk_core::greenlight::node_api=debug,
```
would make sense?

**Review by ok300 — APPROVED** [review]

File: libs/sdk-core/src/breez_services.rs (hunk 2 above)
Comment (JssDWt) [reply libs/sdk-core/src/breez_services.rs:1825]: I thought that would be caught by `breez_sdk_core=debug`, but I'll check

Comment (JssDWt) [reply libs/sdk-core/src/breez_services.rs:1825]: Checked and `breez_sdk_core::greenlight::node_api` logs are included this way

File: libs/sdk-common/src/utils/rest_client.rs (hunk 1 above)
Comment (JssDWt) [reply libs/sdk-common/src/utils/rest_client.rs]: Made the response status a debug log and the raw body a trace log

**Review by roeierez — APPROVED** [review]
LGTM

**Pull request merged**